exampleCourse/questions/element/code/server.py

```diff
@@ -0,0 +1,12 @@
+import pprint
+
+
+def generate(data):
+    complex_object = {
+        (1, 2): "hi",
+        (0, 0): "origin",
+        (2, 2): "another point",
+        (3, 3): "yet another point",
+    }
+
+    data["params"]["my_object_string"] = pprint.pformat(complex_object)
```

Comment:
Could we build the conversion to a string into the serialization part for the tuple & set object? This would negate needing to give a "custom" example.

Reply:
I think doing that has the potential to cause more problems than it's worth. For example, people would need some way to put in custom string conversion options (like indentation size), and could be confusing if objects are being silently converted behind the scenes.